components/homme/src/theta-l/element_ops.F90

```diff
@@ -412,6 +412,7 @@ subroutine set_thermostate(elem,ps,temperature,hvcoord)
   real (kind=real_kind), intent(in) :: temperature(np,np,nlev)
   type (hvcoord_t),     intent(in)  :: hvcoord                      ! hybrid vertical coordinate struct
   real (kind=real_kind), intent(in) :: ps(np,np)
+  real (kind=real_kind), intent(in), optional :: Q1(np,np,nlev)
```

Comment:
Minor comment, I think naming this variable something like "qv" would be clearer just because I associate "Q1" with a heating tendency. At the very least, maybe we can just add a descriptive comment here saying that this should be water vapor.

Reply:
fixed.